scripts/testCoverage.sh

```diff
@@ -16,19 +18,23 @@ generate_cover_data() {
 		echo $pkg
         f="$workdir/$(echo $pkg | tr / -).cover"
 		echo $f
-        godep go test -covermode="$mode" -coverprofile="$f" "$pkg"
+        GOPATH="${TOPDIR}:${TOPDIR}/v:${GOPATH}" godep go test \
+	    -covermode="$mode" \
+	    -coverprofile="$f" "$pkg"
     done
 
     echo "mode: $mode" >"$profile"
     grep -h -v "^mode:" "$workdir"/*.cover >>"$profile"
 }
 
-generate_cover_data $(go list ./... | 
-grep -v github.com/SUSE/cf-usb-sidecar/generated | 
-grep -v github.com/SUSE/cf-usb-sidecar/example | 
-grep -v github.com/SUSE/cf-usb-sidecar/csm-extensions |
-grep -v github.com/SUSE/cf-usb-sidecar/scripts | 
-grep -v github.com/SUSE/cf-usb-sidecar/cmd/catalog-service-manager/handlers | 
-grep -v github.com/SUSE/cf-usb-sidecar/src/api)
+generate_cover_data $(go list ./... | \
+    grep -v github.com/SUSE/cf-usb-sidecar/generated | \
+    grep -v github.com/SUSE/cf-usb-sidecar/example | \
+    grep -v github.com/SUSE/cf-usb-sidecar/csm-extensions | \
+    grep -v github.com/SUSE/cf-usb-sidecar/scripts | \
+    grep -v github.com/SUSE/cf-usb-sidecar/src/github.com/go-swagger | \
+    grep -v github.com/SUSE/cf-usb-sidecar/v/src | \
+    grep -v github.com/SUSE/cf-usb-sidecar/cmd/catalog-service-manager/handlers | \
+    grep -v github.com/SUSE/cf-usb-sidecar/src/api)
```

Comment:
That seems close enough to `go list ./src ./cmd | grep -v /github.com/go-swagger`. The only things that this pulls in (src/api, handlers) don't have tests anyway.

Reply:
Will check/trial

Reply:
Addressed with commit 17a5d57b31950eed6895efbd5fd9cfc2c2496cd4